README.md

```diff
@@ -60,16 +26,25 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Portfolio setup] <a name="about-project">Portfolio setup  of mobile and deskstop version</a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] While you did a wonderful job filling your README file :+1: kindly, please don't forget to remove the section descriptions like:
> Describe the tech stack and include only the relevant sections that apply to your project.